extensions/google-lens/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "google-lens",
+  "title": "Google Lens",
+  "description": "Search the image using Google Lens",
```

Comment:
Maybe some alternatives?

- "Search images with Google Lens"
- "Identify anything in images using Google Lens"
- "Reverse image search powered by Google Lens"